### PR TITLE
fix transform

### PR DESCRIPTION
**Pull request by PangZhi**
This PR is a fix for https://adatao.atlassian.net/browse/BIGAPPS-3898.
The original problem is that in the TranformExpression, you can't have = or ~.

@khangich @nhanitvn @Huandao0812

### CODE REVIEW

File: core/src/main/java/io/ddf/etl/TransformationHandler.java [hunk 1]
```diff
@@ -161,14 +161,23 @@ public static String RToSqlUdf(List<String> RExps, List<String> selectedColumns,
 
     Set<String> newColsInRExp = new HashSet<String>();
     for (String str : RExps) {
-      String[] udf = str.split("[=~](?![^()]*+\\))");
-      String newCol = (udf.length > 1) ?
+      int index = Integer.max(str.indexOf("="), str.indexOf("~"));
```
Comment (khangich) [inline core/src/main/java/io/ddf/etl/TransformationHandler.java]: is it ok to remove other expression like  ! ?

Comment (Huandao0812) [inline core/src/main/java/io/ddf/etl/TransformationHandler.java]: what does this do/

**Comment by khangich** [comment]
UT failed, dont merge yet

**Comment by PangZhi** [comment]
retest this please

**Comment by PangZhi** [comment]
retest this please

**Comment by khangich** [comment]
can we add one more UT for this case ?

**Pull request merged**